app/assets/javascripts/tagging.js

```diff
@@ -1,6 +1,18 @@
 function addTag(tagname, selector) {
 
   selector = selector || '#tagform';
+  if (tagname.indexOf("place") !== -1) {
+    place = tagname.split(":")[1];
+    place.replace("-", " ");
+    geo = geocodeStringAndPan(place);
+
+    if (geo.length > 0) {
```

Comment:
Ok so this block would have to be added in a function as the callback onComplete - or you could move it down inside the onComplete definition on line 102 - make sense?